api-gateway/src/main/resources/application.yml

```
@@ -12,12 +12,55 @@ spring:
+    dbcp2:
+      default-schema: unimal_auth
+
+  data:
+    redis:
+      host: localhost
+      port: 6379
+      password: 6379
+
+custom:
+  jwt:
+    secret-key: ${JWT_SECRET_KEY:and0LXNlY3JldC1rZXktand0LXNlY3JldC1rZXktand0LXNlY3JldC1rZXk=}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)